content/authorization/reference/userscenarios/_index.nb.md

```
@@ -364,6 +367,21 @@ Rett Revisisjon har kjøpt inn Maestro for å rapportere aksjonærregisteroppgav
 
 **Merk** Alternativ løsning for dette vil være at man registerer et system pr kundetype og knytter hver systembruker mot dette. Dette gjør at man unngår extref, men må da håndtere tilsvarende mange klienter. 
 
+
+### 9b.  Nye kunder 
+
+Rett Revisjon har fått nye kunder.  Disse må knyttes til riktig systembruker basert på type klientforhold.
+
+### Forutsetninger
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)